docs/content/docs/install/settings.md

```diff
@@ -186,42 +186,69 @@ A few settings are available to configure this feature:
 
    `<filename>`, `<line>`, `<error>`
 
-### Reporting logs URL to Tekton Dashboard or a custom Dashboard or Console
+### Reporting logs URL to Tekton Dashboard
 
 Pipelines as Code have the ability to automatically detect the OpenShift Console and link the logs of the tasks to the
 public URL of the OpenShift Console. If you are using the Tekton Dashboard, you can configure this feature using the
 `tekton-dashboard-url` setting. Simply set this to your dashboard URL, and the pipelinerun status and tasklog will be
 displayed there.
 
-Alternatively, you can also configure a custom console with the following settings:
+### Reporting to a custom Dashboard/Console
+
+Alternatively, you have the ability to configure the links to go to your custom
+dashboard using the following settings:
```

Comment:
Do we require this doc change here in this PR :thinking: asking because doc change is related to custom console param

Reply:
i rephrase all the time the doc if i find a bug.. no need for new pr or commit...